node/parachain/src/command.rs

```diff
@@ -258,14 +272,27 @@ pub fn run() -> Result<()> {
 				let relay_chain_id = extension.map(|e| e.relay_chain.clone());
 				let para_id = extension.map(|e| e.para_id);
 
-				let polkadot_cli = RelayChainCli::new(
+				let mut polkadot_cli = RelayChainCli::new(
 					config.base_path.as_ref().map(|x| x.path().join("polkadot")),
 					relay_chain_id,
 					[RelayChainCli::executable_name()]
 						.iter()
 						.chain(cli.relaychain_args.iter()),
 				);
 
+				// We override the relay chain id for a few well-known parachain ids.
+				if let Some(chain) = cli.run.base.shared_params.chain {
+					polkadot_cli.base.base.shared_params.chain = Some(
+						match chain.as_str() {
+							"alphanet" => "moonbasealpha",
+							//"moonrock" => "rococo",
+							//"moonriver" => "kusama",
+							//"" | "moonbeam" => "polkadot",
+							other => other,
+						}.to_string()
+					)
+				}
+
```

Comment:
@bkchr Does this seem like a reasonable thing to do? The goal is that collator operators can run a simple command.

```bash
# We want a simple command like this
moonbase --chain moonriver

# Rather than a complex one like this
moonbase --chain moonriver -- --chain kusama
```

Reply:
Check this:
https://github.com/paritytech/cumulus/blob/master/rococo-parachains/src/command.rs#L273-L274

Reply:
Thanks for the pointer. This is much nicer :)